File: C++/1975. Maximum Matrix Sum.cpp
class Solution {
public:
    long long maxMatrixSum(vector<vector<int>>& m) {
     long long c=0;
     int s=INT_MAX,n=0;
     for(int i=0;i<m.size();i++){
        for(int j:m[i]){
            c+=abs(j);
            if(j<0)
            n++;
            s=min(s,abs(j));
        }
     }
     if(n%2==1)
     c-=2*s;
     return c;
    }
};

File: JAVA/1975. Maximum Matrix Sum.java
class Solution {
    public long maxMatrixSum(int[][] m) {
        long c=0;
        int s=Integer.MAX_VALUE,n=0;
        for(int i=0;i<m.length;i++){
            for(int j:m[i]){
                c+=Math.abs(j);
                if(j<0)
                n++;
                s=Math.min(s,Math.abs(j));
            }
        }
        if(n%2==1)
        c-=2*s;
        return c;
    }
}